unity/Assets/0Refactored/scenes/laboratory/Scripts/scrSegmentBuilder.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using GEAR.Serialize;
+
+public class scrSegmentBuilder : MonoBehaviour
+{
+    // #################################################################################################################
+    // Members
+
+    // +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+    // Given Segments
+    public GameObject staticSegementStart;
```

Comment:
Is it required to access these members from outside?
Otherwise please use "[SerializeField] private" instead of "public".